lib/mobiledoc/renderers/0.2.rb

```diff
@@ -67,7 +68,8 @@ def set_attribute(element, prop_name, prop_value)
     end
 
     def create_text_node(text)
-      Nokogiri::XML::Text.new(text.gsub('  ', " #{NBSP}"), doc)
+      text = text.gsub('  ', " #{NBSP}").gsub("\t", EM_SPACE)
```

Comment:
Turns out you can pass a hash to `gsub` to replace different matches with different values:

```suggestion
      text = text.gsub(/(  |\t)/, '  ' => NBSP, "\t" => EM_SPACE)
```

Reply:
Thanks for the suggestion, I will apply it 👍